client/modules/transactions.js

```diff
@@ -38,9 +37,10 @@ function getNextTransactionID(state) {
 
 // Add a new transaction.
 function addTransactionToState(state, action) {
-  const { description, value } = action.transaction;
+  const { categoryId, description, value } = action.transaction;
   const newState = [...state, {
     id: getNextTransactionID(state),
+    categoryId,
```

Comment:
I like that better too but we didn't have object spread enabled in babel at this time. This is what prompted the tickle in Slack the other day about putting that in actually.